Services/Mail/classes/class.ilMailAutoCompleteUserProvider.php

```diff
@@ -197,7 +197,7 @@ protected function getOrderByPart()
      * @param mixed  $a_str
      * @return string
      */
-    protected function getQueryConditionByFieldAndValue($field, $a_str)
+    protected function getQueryConditionByFieldAndValue(string $field, mixed $a_str) : string
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.